network-api/networkapi/wagtailpages/templatetags/card_tags.py

```diff
@@ -9,6 +10,7 @@ def card(image, title, description, link_url, link_label, commitment=None):
         'image': image,
         'title': title,
         'description': description,
+        'description_is_rich_text': BeautifulSoup(description, 'html.parser').p is not None,
```

Comment:
I'm not sure this is a safe way to check this: where is `description` coming from? Because rich text could also be `<ul>...</ul>` etc. so we don't want to hardcode that only things with `<p>` are rich text.

Reply:
Ah agreed that this is not very future proof. I only took CTA's `subhead` into consideration (passed to template as `description`).

```py
subhead = RichTextField(
        features=[
            'bold', 'italic', 'link',
        ],
        blank=True,
    )
```

Will update PR.